src/utils/castBackend.ts

```diff
@@ -0,0 +1,15 @@
+import { ExternalLibraryName, Library } from '../components/assessment/assessmentShape'
+
+export const castLibrary = (lib: any): Library => ({
```

Comment:
Good to document differences in frontend and backend data, like [here](https://github.com/source-academy/cadet-frontend/blob/c61fd6009f480bccc4ed06470a364e5ae9c98b4b/src/sagas/backend.ts#L308).